libs/dmlf/examples/boston_housing_multiprocess_distributed_learning/main.cpp

```diff
@@ -76,8 +76,8 @@ int main(int argc, char **argv)
 
   // Load data
 
-  TensorType data_tensor  = fetch::ml::dataloaders::ReadCSV<TensorType>(data_file).Transpose();
-  TensorType label_tensor = fetch::ml::dataloaders::ReadCSV<TensorType>(labels_file).Transpose();
+  TensorType data_tensor  = fetch::math::utilities::ReadCSV<TensorType>(data_file).Transpose();
+  TensorType label_tensor = fetch::math::utilities::ReadCSV<TensorType>(labels_file).Transpose();
```

Comment:
This transposes in both versions - now that ReadCSV transposes by default does this still  have the correct behaviour? The same comment applies to all the other dmlf and ml examples that use this.

Reply:
good catch. shouldn't be a transpose anymore